backend/database/schema.sql

```diff
@@ -1,4 +1,74 @@
-create table item (
-  id int unsigned primary key auto_increment not null,
-  title varchar(255) not null
+create table user (
+  id int primary key auto_increment not null,
+  lastname varchar(50) not null,
+  firstname varchar(50) not null,
+  email varchar(255) not null,
+  password_hash varchar(32) not null
 );
+create table artist (
+  id int primary key auto_increment not null,
+  biography text,
+  user_id int,
+  constraint fk_artist_user
+  foreign key (user_id)
+  references user(id)
+);
+create table administrator (
+  id int primary key auto_increment not null,
+  user_id int,
+  constraint fk_artist_user
+  foreign key (user_id)
+  references user(id)
+);
+create table logged_user (
+  id int primary key auto_increment not null,
+  user_id int,
+  constraint fk_artist_user
+  foreign key (user_id)
+  references user(id)
+);
+create table artwork (
+  id int primary key auto_increment not null,
+  title varchar(80) not null,
+  technique varchar(50) not null,
+  artwork_year int not null,
+  format varchar(50) not null,
+  source varchar(150) not null,
+  alt varchar(80) not null,
+  user_id_ar int not null,
+  constraint fk_artwork_artist
+  foreign key (user_id_ar)
+  references artist(user_id)
+);
+create table management (
+  user_id_ad int not null,
```

Comment:
id is missing

Reply:
There is a many-to-many relation between admin and artwork tables. Management doesn't need an id, it's an intermediary table